integration/commands_administration_test.go

```diff
@@ -299,126 +298,186 @@ func TestCommandsAdministrationGetParameter(t *testing.T) {
-				{"ok", float64(1)},
-			},
-		},
-		"DBStatsWithScale": {
-			command: bson.D{{"dbStats", int32(1)}, {"scale", float64(1_000)}},
-			response: bson.D{
-				{"db", collection.Database().Name()},
-				{"collections", int32(1)},
-				{"views", int32(0)},
-				{"objects", int32(43)},
-				{"avgObjSize", 433.0},
-				{"dataSize", 8.192},
-				{"indexes", int32(0)},
-				{"indexSize", float64(0)},
-				{"totalSize", 16.384},
-				{"scaleFactor", float64(1_000)},
-				{"ok", float64(1)},
-			},
-		},
-		"ServerStatus": {
-			command: bson.D{{"serverStatus", int32(1)}},
-			response: bson.D{
-				{"host", ""},
-				{"version", "5.0.42"},
-				{"process", "handlers.test"},
-				{"pid", int64(0)},
-				{"uptime", int64(0)},
-				{"uptimeMillis", int64(0)},
-				{"uptimeEstimate", int64(0)},
-				{"localTime", primitive.DateTime(time.Now().Unix())},
-				{"catalogStats", bson.D{
-					{"collections", int32(1)},
-					{"capped", int32(0)},
-					{"timeseries", int32(0)},
-					{"views", int32(0)},
-					{"internalCollections", int32(0)},
-					{"internalViews", int32(0)},
-				}},
-				{"freeMonitoring", bson.D{{"state", "disabled"}}},
-				{"ok", float64(1)},
-			},
-		},
-	} {
-		name, tc := name, tc
-		t.Run(name, func(t *testing.T) {
-			t.Parallel()
+	var actual bson.D
+	command := bson.D{{"buildInfo", int32(1)}}
+	err := collection.Database().RunCommand(ctx, command).Decode(&actual)
+	require.NoError(t, err)
 
-			var actual bson.D
-			err := collection.Database().RunCommand(ctx, tc.command).Decode(&actual)
-			require.NoError(t, err)
+	doc := ConvertDocument(t, actual)
 
-			AssertEqualDocuments(t, tc.response, actual)
-		})
-	}
+	assert.Equal(t, float64(1), must.NotFail(doc.Get("ok")))
+	assert.Regexp(t, `^5\.0\.`, must.NotFail(doc.Get("version")))
+	assert.NotEmpty(t, must.NotFail(doc.Get("gitVersion")))
+
+	_, ok := must.NotFail(doc.Get("modules")).(*types.Array)
+	assert.True(t, ok)
+
+	assert.Equal(t, "deprecated", must.NotFail(doc.Get("sysInfo")))
+
+	versionArray, ok := must.NotFail(doc.Get("versionArray")).(*types.Array)
+	assert.True(t, ok)
+	assert.Equal(t, int32(5), must.NotFail(versionArray.Get(0)))
+	assert.Equal(t, int32(0), must.NotFail(versionArray.Get(1)))
+
+	assert.Equal(t, int32(strconv.IntSize), must.NotFail(doc.Get("bits")))
+	assert.False(t, must.NotFail(doc.Get("debug")).(bool))
+
+	assert.Equal(t, int32(16777216), must.NotFail(doc.Get("maxBsonObjectSize")))
+	_, ok = must.NotFail(doc.Get("buildEnvironment")).(*types.Document)
+	assert.True(t, ok)
+}
+
+func TestCommandsAdministrationCollStats(t *testing.T) {
+	t.Parallel()
+	ctx, collection := setup(t)
+
+	var actual bson.D
+	command := bson.D{{"collStats", collection.Name()}}
+	err := collection.Database().RunCommand(ctx, command).Decode(&actual)
+	require.NoError(t, err)
+
+	doc := ConvertDocument(t, actual)
+	assert.Equal(t, float64(1), must.NotFail(doc.Get("ok")))
+	assert.Equal(t, collection.Database().Name()+"."+collection.Name(), must.NotFail(doc.Get("ns")))
```

Comment:
So, in the tests in this PR, we use empty tables, right? I see we compare almost all the values with zeros... Does it make sense to have cases when things like `storageSize` should be greater than zero?

Reply:
> So, in the tests in this PR, we use empty tables, right?

right.

> I see we compare almost all the values with zeros... Does it make sense to have cases when things like `storageSize` should be greater than zero?

Do you mean creating a table, adding data there, and testing whether storage size in the response increases?

Reply:
@seeforschauer yes, maybe something like this. I'm not sure if it's reasonable and doable, but it seems to be important to check not only for the "init" values :)

Reply:
>  I'm not sure if it's reasonable and doable, but it seems to be important to check not only for the "init" values :)

It's reasonable and doable. 
Here in the task, I need just to enable the previously working tests. 

I suggest creating a separate PR for adding that another test since I can find something and change the code lot  ;-)

Reply:
> I guess then we need a separate issue too? I'm ok with it.

On empty tables - added some tests for tables with data.

Also created #727 for the error `numObjects`